notebooks/2.1.1-he-rain-analysis.ipynb

```diff
@@ -72,22 +72,31 @@
    "metadata": {},
```

Comment:
<p>same as above: second and third are not consensus values (i didnt calculate consesus for altitude/distance) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/DataKind-DC/audubon-cbc/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='112'/>

Reply:
<p>Fixed in Rain, snow, and snowfall notebooks</p>